.build/deps.mk

```diff
@@ -4,8 +4,8 @@ libdeps:
 	$(ECHO_V)glide --version 2>/dev/null ||	go get -v github.com/Masterminds/glide
 	$(ECHO_V)glide install
 
-.PHONY: dependencies
-dependencies: libdeps
+.PHONY: deps
```

Comment:
`make deps` is in examples, and `make dependencies` is annoying to type, especially since it doesn't autocomplete from the included .mk file :(